internal/store/organizations.go

```diff
@@ -29,7 +29,7 @@ type OrganizationLister interface {
 }
 
 type OrganizationDescriber interface {
-	Organization(string) (*atlas.Organization, error)
+	Organization(string) (interface{}, error)
```

Comment:
I understand the need for `interface{}` here, but this to me feels like splitting the stores is the right way of approaching 💔